localstack/services/events/events_starter.py

```diff
@@ -44,10 +44,21 @@ def send_event_to_firehose(event, arn):
         Record={'Data': to_bytes(json.dumps(event))})
 
 
+def filter_event_with_target_input_path(target, event):
+    input_path = target.get('InputPath')
+    if input_path:
+        paths = input_path.split('.')
```

Comment:
Instead of manual splitting/traversal here, can we instead utilize the JSONPath extraction util function here: https://github.com/localstack/localstack/blob/master/localstack/utils/common.py#L778-L783